src/contracts/interfaces/IIndexRegistry.sol

```diff
@@ -86,5 +86,5 @@ interface IIndexRegistry is IRegistry {
     function totalOperatorsForQuorum(uint8 quorumNumber) external view returns (uint32);
 
     /// @notice Returns an ordered list of operators of the services for the given `quorumNumber` at the given `blockNumber`
-    function getOperatorListForQuorumAtBlockNumber(uint8 quorumNumber, uint32 blockNumber) external view returns (bytes32[] memory);
+    function getOperatorListForQuorumAtBlockNumber(uint8 quorumNumber, uint32 blockNumber) external returns (bytes32[] memory);
```

Comment:
likewise, this should be restored to 'view' prior to merging

Reply:
https://github.com/Layr-Labs/eigenlayer-contracts/pull/88/commits/8888f2a31ff9e13bca8370846c35f1c48b45d48f